### ISSUE TITLE
Can you make your code work with Python 3?

### ISSUE THREAD

**Issue by sarimmehdi** [mileyan/pseudo_lidar#36]
Hi. To run your code and get the proper depth map we have to switch to Python 2 otherwise the depth map is completely incorrect. Can you make this compatible with Python 3? Thanks!

**Comment by jarvis-huang** [comment]
The official [PSMNet](https://github.com/JiaRenChang/PSMNet) was already ported to Python 3. So you should be able to compute disparity map already. What pseudo-lidar does is simply converting the disparity into depth map `generate_lidar.py`. I believe this code should already work in Python 2 or maybe with minimal modification. After all, it has very few lines of code.

**Comment by Cli98** [comment]
> The official [PSMNet](https://github.com/JiaRenChang/PSMNet) was already ported to Python 3. So you should be able to compute disparity map already. What pseudo-lidar does is simply converting the disparity into depth map `generate_lidar.py`. I believe this code should already work in Python 2 or maybe with minimal modification. After all, it has very few lines of code.

@jarvis-huang 
This is wrong sir. The code in this repo only support python2. They do update their code base but not reflected here.
